src/afl-fuzz.c

```diff
@@ -2554,6 +2577,109 @@ int main(int argc, char **argv_orig, char **envp) {
 
   }
 
+  san_abstraction = getenv("AFL_SAN_ABSTRACTION");
```

Comment:
could you move this into our env walking in afl-fuzz-stats pls?

Reply:
This has been included in `afl_environment_variables`

Reply:
that is not what I meant. get rid of the getenv here and move it over in the env parsing in afl-fuzz-stats.c :)